README.md

```diff
@@ -1074,6 +1074,61 @@ fga query **expand** <relation> <object> --store-id=<store-id> [--model-id=<mode
 }
 ```
 
+##### List Users
+
+###### Command
+fga query **list-users** --object <object> --relation <relation> --user-filter <user-filter> [--contextual-tuple "<user> <relation> <object>"]* --store-id=<store-id> [--model-id=<model-id>]
+
+###### Parameters
+* `--store-id`: Specifies the store id
+* `--object`: Specifies the object to list users for
+* `--relation`: Specifies the relation to search on
+* `--user-filter`: Specifies the type or userset to filter with
```

Comment:
#311 does state `--user_filter` but given that kebab case is our standard in the CLI I went with that